rootfs/etc/apache2/conf-enabled/servername.conf

```diff
@@ -0,0 +1 @@
+ServerName localhost
```

Comment:
I think you can also add this to security.conf:

```
ServerTokens Prod
ServerSignature Off
TraceEnable Off
```

Reply:
done